lib/typhoeus/response/header.rb

```diff
@@ -47,7 +47,7 @@ def parse
       # @return [ void ]
       def process_line(header)
         key, value = header.split(':', 2)
-        process_pair(key.strip, value.strip.gsub(/\r?\n\s*/, ' '))
+        process_pair(key.strip, (value ? value.strip.gsub(/\r?\n\s*/, ' ') : nil))
```

Comment:
@jarthod thank you for your work! I am not sure if we should use empty string or nil.